### ISSUE TITLE
Add your own intent

### ISSUE THREAD

**Issue by epicfaace** [stanfordmsp/bot-framework-tutorial#2]
# Add your own intent
Time to add your own intent! Click on "Create new intent" to create a new intent.

In this example, we are creating a new intent which lets the bot know if you want to drop out of school.

1. Click on "Create intent" and type "Dropout" as the intent name.
![image](https://user-images.githubusercontent.com/1689183/38036918-ce6cddfa-3275-11e8-8f31-dba7872cdf94.png)

1. Enter several examples of what the user may say in order for the bot to recognize the "Dropout" intent.
![image](https://user-images.githubusercontent.com/1689183/38037164-3fcf5bd0-3276-11e8-8571-de2ec7432d17.png)

After you've typed a few phrases, the screen should look like this:

![image](https://user-images.githubusercontent.com/1689183/38037106-27089a44-3276-11e8-9043-7690e5cfea17.png)

1. Finally, click on the "Train" button on the top right to train your bot. Then click on the "Test" button to test your intent. Type in a few phrases that have the same meaning as dropping out of school and see what the bot classifies it as. You can see that the bot has varying levels of success with classifying intents; this may improve with better training data.

![image](https://user-images.githubusercontent.com/1689183/38037287-8e618fe8-3276-11e8-9a10-3bacd51b6d9a.png)
